emojify.el

```diff
@@ -1837,7 +1842,12 @@ completion UI to display properly emojis."
   (let* ((emojify-minibuffer-reading-emojis-p t)
          (line-spacing 7)
          (completion-ignore-case t)
-         (candidates (emojify--get-completing-read-candidates))
+         ;; Ido can only accept a list so grab the keys of the hashmap
+         (candidates (let ((keys ()))
```

Comment:
Can you update `emojify--get-completing-read-candidates` to return a list instead of a hash-table, that way the final list is cached

Reply:
Sure thing! I wasn't sure if you were using a hash table for any particular reason and didn't want to wipe out a data-structure that might hold a ton of info. you might use elsewhere.

Reply:
Makes sense, I verified the same thing and realized that I was not using the data anywhere so I guess it safe to remove it